Solutions/Cybersixgill-Actionable-Alerts/Playbooks/DeleteCybersixgillAlert/readme.md

```diff
@@ -0,0 +1,28 @@
+# DeleteCybersixgillAlert
+author: Loginsoft
+
+This playbook will delete Actionable alerts in Cybersixgill Portal. When incident is deleted in Microsoft Sentinel, playbook will run and delete Actionable alerts from Portal 
+
+# Prerequisites
+We will need the following data to do one time setup
+
+1. Cybersixgill Client ID (client_id)
+2. Cybersixgill Client Secret (client_secret)
+

```

Comment:
Please provide more info. On how customer can fetch the above-mentioned details from  Cybersixgill  UI /portal

Reply:
Sure